tensorflow/python/ops/nn_grad.py

```diff
@@ -897,21 +897,28 @@ def _BaseFusedBatchNormGrad(op, version, *grad):
     if data_format == b"NCHW":
       x = array_ops.transpose(x, [0, 2, 3, 1])
       grad_y = array_ops.transpose(grad_y, [0, 2, 3, 1])
+    elif data_format == b"NCDHW":
+      x = array_ops.transpose(x, [0, 2, 3, 4, 1])
+      grad_y = array_ops.transpose(grad_y, [0, 2, 3, 4, 1])
+    target_data_format = ("NHWC" if data_format == b"NCHW" or
```

Comment:
the data_format check should be `data_format in (b"NCHW", b"NHWC")`

Reply:
Done.